renderer/lib/cast.js

```diff
@@ -21,50 +22,102 @@ module.exports = {
 // Callback to notify module users when state has changed
 var update
 
-function init (callback) {
-  update = callback
-
-  // Start polling Chromecast or Airplay, whenever we're connected
-  setInterval(() => pollCastStatus(state), 1000)
-
-  // Listen for devices: Chromecast and Airplay
-  chromecasts.on('update', function (player) {
-    state.devices.chromecast = player
-    addChromecastEvents()
-  })
+// chromecast player implementation
+function chromecastPlayer (player) {
```

Comment:
Maybe `ChromecastPlayer`, and then we can say

```js
this.player = player
this.open = open
...
```

...and create one with `new ChromecastPlayer`

We might even make a `Player` and have them prototype inherit from that